neptune/new/internal/backends/hosted_neptune_backend.py

```diff
@@ -540,8 +540,10 @@ def _execute_operations(self,
             return [MetadataInconsistency(err.errorDescription) for err in result]
         except HTTPNotFound as e:
             raise RunUUIDNotFound(run_id=run_id) from e
-        except HTTPUnprocessableEntity:
-            raise NeptuneStorageLimitException()
+        except (HTTPPaymentRequired, HTTPUnprocessableEntity) as e:
+            raise NeptuneLimitExceedException(
+                reason=e.response.json().get("message", "Maximum storage limit reached")
```

Comment:
Have you tested compatibility with old and new api on real Neptune instance? Are you sure there is `message` field in response?

Reply:
I thought that it was not implemented yet on the backend side currently (passing exact reason). It was tested on the current "stage" for storage limit. It was created based on legacy structure from https://github.com/neptune-ai/neptune-client/blob/2801c717b5c8ad983715421f2ce272be9b5816a9/neptune/internal/channels/channels_values_sender.py#L161. I've updated `"message"` to `"title"` which is based on current error response structure. What do you mean by old vs new API?

Reply:
Old - current response from leaderbaord.
New - what is under development on backend side right now.